components/brave_stats/browser/brave_stats_updater_util.cc

```diff
@@ -30,7 +30,10 @@ std::string GetPlatformIdentifier() {
   else
     return "winx64-bc";
 #elif defined(OS_MAC)
-  return "osx-bc";
+  if (base::SysInfo::CPUModelName().find("Intel") != std::string::npos)
+    return "osx-bc";
+  else
+    return "osx-arm64-bc";
```

Comment:
This new platform identifier (`osx-arm64-bc`) will need to be supported on the server as well, from what I understand.

cc: @aekeus

Reply:
https://github.com/brave-intl/go-vault/issues/55